jaqpotpy/descriptors/tests/test_mordred_descriptors.py

```diff
@@ -14,55 +14,36 @@ def setUp(self):
         Set up tests.
         """
         from rdkit import Chem
-        smiles = 'CC(=O)OC1=CC=CC=C1C(=O)O'
-        self.mol = Chem.MolFromSmiles(smiles)
-        smiles = 'CCCC(=O)OC1=CC=CC=C1C(=O)O'
-        self.mol2 = Chem.MolFromSmiles(smiles)
+        self.smiles1 = 'CC(=O)OC1=CC=CC=C1C(=O)O'
+        self.mol = Chem.MolFromSmiles(self.smiles1)
+        self.smiles2 = 'CCCC(=O)OC1=CC=CC=C1C(=O)O'
+        self.mol2 = Chem.MolFromSmiles(self.smiles2)
+        self.featurizer = MordredDescriptors()
 
     def test_mordred_descriptors(self):
         """
-        Test simple descriptors.
+        Test featurize using mols
         """
-        featurizer = MordredDescriptors()
-        descriptors = featurizer([self.mol])
-        assert descriptors.shape == (1, 1613)
-        assert np.allclose(descriptors[0][0:3],
-                           np.array([9.54906713, 9.03919229, 1.0]))
+        descriptors = self.featurizer([self.mol, self.mol2])
+        assert descriptors.shape == (2, 1613), 'Wrong shape'
+        assert isinstance(descriptors[0][1], (int, float, np.number)), 'The value is not numeric'
 
     def test_mordred_descriptors_dataframe(self):
         """
-        Test simple descriptors.
-        """
-        featurizer = MordredDescriptors()
-        descriptors = featurizer.featurize_dataframe([self.mol])
-        assert descriptors.shape == (1, 1613)
-        assert descriptors.iat[0, 0] == 9.54906712535007
-        assert descriptors.iat[0, 1] == 9.039192285773227
-        assert descriptors.iat[0, 2] == 1.0
-
-    def test_mordred_descriptors_dataframe_two_row(self):
-        """
-        Test simple descriptors.
+        Test featurize_dataframe using mols
         """
         featurizer = MordredDescriptors()
```

Comment:
featurizer = MordredDescriptors() has been defined as a class attribute so this should be removed now